### ISSUE TITLE
你好能否上传requirements文件

### ISSUE THREAD

**Issue by zxt6174** [zhangy0822/USER#3]
请问您使用的环境和用到的包

**Comment by zxt6174** [comment]
请问论文中的CGE模块在哪个文件里，

**Comment by xiaoxiaoxiaofufu** [comment]
> 请问论文中的CGE模块在哪个文件里，

请问你解决了吗，我也没找到

**Comment by zhangy0822** [comment]
> 请问您使用的环境和用到的包

Our method has no special dependency on the package, I will upload a usable environment for you.

**Comment by zhangy0822** [comment]
> 请问论文中的CGE模块在哪个文件里，

The CGE module is easy to implement, but requires you to access CLIP model from https://github.com/openai/CLIP and extract the features in advance. Specific implementation of CGE module, including nn.BatchNorm1d (512), nn.Linear (512, embed_size) nn.BatchNorm1d (embed_size), nn.Linear (embed_size, embed_size) ( Activation functions need to be added.). You can tailor it to your situation.
